tests/jax/components/executing/base_test.py

```diff
@@ -73,7 +73,23 @@ def test_on_execution_init_start(
     executor_init = ExecutorInit(config=dummy_config)
     executor_init.on_execution_init_start(executor=mock_executor)
 
-    assert mock_executor._interval == dummy_config.interval  # type: ignore
+    assert mock_executor.store._evaluation_interval == None  # type: ignore # noqa: E501
+
+
+def test_on_execution_init_start_with_evaluator(
+    mock_executor: Executor, dummy_config: ExecutorInitConfig
+) -> None:
+    """Test on_execution_init_start method from ExecutorInit
+
+    Args:
+        mock_executor: Executor
+        dummy_config: ExecutorInitConfig
+    """
+    mock_executor.store.is_evaluator = True
+    executor_init = ExecutorInit(config=dummy_config)
+    executor_init.on_execution_init_start(executor=mock_executor)
+
+    assert mock_executor.store._evaluation_interval == dummy_config.evaluation_interval  # type: ignore # noqa: E501
```

Comment:
please add an assertion to test duration being set also

Reply:
Done :)